pkg/internal/checkup/trex-client/trex-client-types.go

```diff
@@ -1,61 +1,65 @@
 package trexclient
 
 type GlobalStats struct {
-	ID      string `json:"id"`
```

Comment:
Please consider moving this change to a follow-up PR.
I think it is doing more than the stated title of "Move trex console to package".

Reply:
How is it doing more ? I'm just moving files to a new package.